textile/overrides/sales_invoice_hooks.js

```diff
@@ -35,4 +40,228 @@ frappe.ui.form.on("Printed Fabric Detail", {
 		textile.set_printed_fabric_rate(frm, row);
 		frm.cscript.calculate_taxes_and_totals();
 	},
+
+	before_printed_fabrics_remove: function(frm, cdt, cdn) {
+		var printed_fabrics_rows_selected = frm.get_selected().printed_fabrics
```

Comment:
That's not a good idea... rows can be remove without selecting as well, try a different approach